src/components/common/Team/HighlightTeam.tsx

```diff
@@ -1,46 +1,44 @@
 import { useEffect, useState } from 'react';
-import { useRouter } from 'next/router';
+import Link from 'next/link';
+import { io } from 'socket.io-client';
 import TeamDataCard from './TeamDataCard';
 import { httpClient } from '@/apis/httpClient';
+import { GroupBuyingData } from '@/types/apis/groupBuying';
 import styles from './HighlightTeam.module.scss';
 
-interface GroupUser {
-  nickname: string;
+interface HighlightTeamProps {
+  productId: number;
 }
 
-interface GroupBuyingData {
-  id: number;
-  status: number;
-  groupUsers: GroupUser[];
-}
-
-export default function HighlightTeam({ productId }: any) {
-  const router = useRouter();
-
+export default function HighlightTeam({ productId }: HighlightTeamProps) {
   const [teamData, setTeamData] = useState<GroupBuyingData[]>([]);
 
+  const socket = io('http://ec2-13-209-80-36.ap-northeast-2.compute.amazonaws.com/');
```

Comment:
💊 내일 테스트 끝나면 env에 등록한 url로 바꿔주시면 될 거 같아요~

```suggestion
  const socket = io(`${process.env.NEXT_PUBLIC_API_BASE_URL}`);
```

Reply:
[271385e](https://github.com/Together-3team/petFrontend/pull/249/commits/271385ec97805cd9804691ce36a9be84bf196c4c)

수정했습니다 !